roles/observe_chargeback/README.md

```diff
@@ -0,0 +1,49 @@
+observe_chargeback
```

Comment:
Please rename the role to be consistent with the other ones in the repo.

i.e. telemetry_chargeback for the roles that test rhoso